.github/workflows/pre_release_version_bump.yml

```diff
@@ -0,0 +1,30 @@
+name: Manually Triggered Version Bumps
+
+on:
+  push:
+  #workflow_dispatch:
+
+jobs:
+  pre_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run pre-release version bump
+        uses: PennyLaneAI/automation/version_bump_action@v1.0
+        with:
+          version_path: "./pennylane_aqt/_version.py"
+          changelog_path: "./CHANGELOG.md"
+          release_status: "pre_release"
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: pre release version bump
+          title: Version Bump
+          body: updated changelog and _version.py
+          branch: pre-release-version-bump
+          reviewers: albi3ro
```

Comment:
These actions will create PRs for version bumps and assign @albi3ro as reviewer. We will test it for the next few releases and if all goes well can change it to allow for auto merging as well 👍🏼